ska_conda/build_order.txt

```diff
@@ -3,22 +3,29 @@ Ska.File
 pyyaks
 ska_path
 testr
-Ska.tdb
+tables3_api
 Chandra.Time
+Ska.Matplotlib
+Ska.tdb
 Ska.ParseCM
+parse_cm
 Ska.DBI
 Ska.ftp
 Ska.Numpy
 Quaternion
-Ska.engarchive
-kadi
-Ska.Matplotlib
 Ska.quatutil
 Ska.Sun
+Ska.engarchive
+agasc
 Chandra.Maneuver
-cmd_states
+kadi
+Chandra.cmd_states
 xija
 maude
-#agasc
-#chandra_aca
-ska
\ No newline at end of file
+chandra_aca
+Ska.arc5gl
+Ska.astro
+mica
+cxotime
+annie
+ska-flight
```

Comment:
The interface doc specifies `ska3-flight`.  I went back and forth about whether to try being consistent with `ska3` branding instead of `ska` to make this distinction clear to ourselves and end-users.  In a few years the `3` might seem superfluous, but for now when we say "Ska flight" that is really Ska2 flight, so probably worth being rigorous in here.